app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -96,31 +96,24 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
         }
         setContentView(R.layout.activity_main)
 
-        val isTheFirstLaunch = settings.getAppLaunchCount() == 0
-        if (isTheFirstLaunch) {
-            setSplashScreenPreDrawListener()
-        }
-
         startupPathProvider.attachOnActivityOnCreate(lifecycle, intent)
         startupTypeTelemetry = StartupTypeTelemetry(components.startupStateProvider, startupPathProvider).apply {
             attachOnMainActivityOnCreate(lifecycle)
         }
 
-        val intent = SafeIntent(intent)
-
-        // The performance check was added after the shouldShowFirstRun to take as much of the
-        // code path as possible
-        if (settings.isFirstRun() &&
-            !Performance.processIntentIfPerformanceTest(intent, this)
```

Comment:
@Amejia481 @ionutbedregeanu We removed the onboarding workaround for performance tests?